htdocs/compta/bank/bankentries.php

```diff
@@ -651,7 +651,7 @@
 	    print '</td><td>';
 	    print '<input name="num_chq" class="flat" type="text" size="4" value="'.GETPOST("num_chq").'"></td>';
 	    print '<td colspan="2">';
-	    print '<input name="label" class="flat" type="text" size="24"  value="'.GETPOST("label").'">';
+	    print '<input name="label" class="flat" type="text" size="40"  value="'.GETPOST("label").'">';
```

Comment:
Using of size="xx" into search field are no more allowed for coming PR.
We must now use class="abcwidthxxx"
when abc is 'min' of 'max' and xxx is 50, 100, 150, 200, depending on need (see css file for complete list of min/maxwidth known.